src/index.html

```diff
@@ -9,6 +9,6 @@
 <body>
     <div id = "root"></div>
 
-    <script src="index.bundle.js"></script>
+    <!-- <script src="index.bundle.js"></script> -->
```

Comment:
Why did you comment this out?

Reply:
we commented that line because it was blocking the DOM to be manipulated